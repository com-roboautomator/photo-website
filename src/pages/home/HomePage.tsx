import {createStyles, WithStyles, withStyles} from '@material-ui/styles'
import React from 'react'
import HomeImageSlider from '../components/carousel/HomeImageSlider'
import PageTitle from '../components/text/PageTitle'
import {collection} from 'src/assets/data/ImageDataStructure'
import {settings} from '../../assets/settings.json'

interface HomeProps extends WithStyles<typeof styles> {}

interface HomeState {
    data: collection[]
}

class Home extends React.Component<HomeProps, HomeState> {
    constructor(props: HomeProps) {
        super(props)
        this.state = {
            data: [],
        }
    }

    getWindowDimensions = () => {
        const {innerWidth: width, innerHeight: height} = window
        return {
            width,
            height,
        }
    }

    componentDidMount() {
        console.log(settings.domain + settings.port + '/collection')
        fetch(settings.domain + settings.port + '/collection', {
            method: 'get',
            headers: {
                'Content-Type': 'application/json',
            },
        })
            .then((response) => response.json())
            .then((response) => {
                this.setState({
                    data: response,
                })
                console.log(this.state.data)
            })
    }

    render() {
        const classes = this.props.classes

        if (this.state.data.length === 0) {
            console.log('returning')
            return <div />
        }
        return (
            <main>
                <div data-testid={'Home-Page'} className={classes.container}>
                    <PageTitle
                        title="HOME"
                        height={40}
                        signature={true}
                        src="https://doneux-website.s3.eu-west-2.amazonaws.com/fd1f733e-2cdf-46ec-b1eb-3e5e55a98a93.jpg"
                    />
                    {console.log(this.state.data)}
                    <div className={classes.wapper}>
                        <HomeImageSlider
                            height={
                                window.innerHeight / 5 < 400
                                    ? 300
                                    : window.innerHeight / 5
                            }
                            data={this.state.data}
                        />
                    </div>
                </div>
            </main>
        )
    }
}

const styles = () =>
    createStyles({
        container: {
            zIndex: 0,
            position: 'relative',
            width: '100%',
        },
        wapper: {
            marginLeft: '80px',
            marginRight: '80px',
            marginTop: '5vh',
        },
    })

export default withStyles(styles)(Home)
