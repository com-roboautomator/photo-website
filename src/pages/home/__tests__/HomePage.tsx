import HomePage from '../HomePage'
import renderer from 'react-test-renderer'
import {render} from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'

describe('Home Page', () => {
    it('should match snapshot', () => {
        const navigationBarRender = renderer.create(<HomePage />).toJSON()
        expect(navigationBarRender).toMatchSnapshot()
    })

    it('should render correctly', () => {
        const {getByTestId} = render(<HomePage />)

        expect(getByTestId('PageTitle-Container')).toBeInTheDocument()
        expect(getByTestId('ImageSlider-Container')).toBeInTheDocument()
    })
})
