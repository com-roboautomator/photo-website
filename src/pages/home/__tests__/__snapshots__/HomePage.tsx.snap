// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should match snapshot 1`] = `
<main>
  <div
    className="Home-container-1"
    data-testid="Home-Page"
  >
    <div
      className="PageTitle-container-3"
      data-testid="PageTitle-Container"
    >
      <img
        alt=""
        className="PageTitle-image_loading-5"
        data-testid="PageTitle-Image"
        onLoad={[Function]}
        src="https://picsum.photos/id/1040/1500/200"
        style={
          Object {
            "height": "45vh",
          }
        }
      />
      <div
        className="PageTitle-title_loading-6"
        data-testid="PageTitle-Text"
      >
        HOME
      </div>
      <div
        className="PageTitle-signature-8"
      >
        <img
          alt=""
          data-testid="PageTitle-Signature"
          src="Logo.png"
          width="25%"
        />
      </div>
    </div>
    <div
      className="Home-wapper-2"
    >
      <main
        className="ImageSlider-container-9"
      >
        <div
          className="ImageSlider-wrapper-10"
          data-testid="ImageSlider-Container"
        >
          <div
            className="HomeImageSliderButtonWrapper-button_wrapper-13"
            data-testid="ImageSlider-button-wrapper"
            style={
              Object {
                "background": "linear-gradient(to right, white, transparent 30%), linear-gradient(to left, white, transparent 30%)",
              }
            }
          >
            <div
              className="HomeImageSliderButtonWrapper-button_wrapper-13"
              data-testid="Buttom-Wrapper"
            >
              <div
                className="HomeImageSliderButtonWrapper-arrow-14"
              >
                <div
                  className="Arrow-container-15"
                >
                  <button
                    className="MuiButtonBase-root MuiButton-root MuiButton-text Mui-disabled Mui-disabled"
                    data-testid="Arrow-Button-Left"
                    disabled={true}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={-1}
                    type="button"
                  >
                    <span
                      className="MuiButton-label"
                    >
                      <img
                        alt="nata"
                        className="Arrow-left-16"
                        data-testid="Arrow-Button-Image"
                        src="arrow-disabled.png"
                      />
                    </span>
                  </button>
                </div>
              </div>
              <div
                className="HomeImageSliderButtonWrapper-arrow-14"
              >
                <div
                  className="Arrow-container-15"
                >
                  <button
                    className="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-testid="Arrow-Button-Right"
                    disabled={false}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                    type="button"
                  >
                    <span
                      className="MuiButton-label"
                    >
                      <img
                        alt="nata"
                        className="Arrow-right-17"
                        data-testid="Arrow-Button-Image"
                        src="arrow-enabled.png"
                      />
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
          <div
            className="ImageSlider-card_slider-11"
            data-testid="ImageSlider-Card-Slider"
            onWheel={[Function]}
            style={
              Object {
                "height": "270px",
              }
            }
          >
            <div
              className="ImageSlider-card_slider_wrapper-12"
              style={
                Object {
                  "transform": "translateX(-0%)",
                }
              }
            >
              <div
                className="CollectionCard-spacing_selected-18"
                data-testid="Home-Collection-Card-Nature Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/befee15c-c770-4450-b71f-f20f820b3549"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/1018/800"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Nature Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#47742c",
                            }
                          }
                        >
                          Nature
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-Urban Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/16bdf4fe-692d-4e11-8fd4-9ea37e3fc34d"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/101/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Urban Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#323232",
                            }
                          }
                        >
                          Urban
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-Animal Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/2c615617-6372-4042-bce5-a97910a51b5a"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/1024/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Animal Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#323232",
                            }
                          }
                        >
                          Nature
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-Mechanical Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/3961f578-14ea-46aa-9b74-7f908fcd532b"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/111/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Mechanical Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#50b1d6",
                            }
                          }
                        >
                          Technology
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-Party Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/37f92b19-2ea1-4742-b97b-81ed97e2e9c3"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/158/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Party Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#323232",
                            }
                          }
                        >
                          Default
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-Second Nature Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/6dcdba12-f48c-46cf-b1f6-07e8d1c5ed2b"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/1018/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Second Nature Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#323232",
                            }
                          }
                        >
                          Default
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-Second Urban Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/2da159b9-1632-4ff7-9bc0-5473f9523ab7"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/101/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Second Urban Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#323232",
                            }
                          }
                        >
                          Default
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-Second Animal Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/fe5bff7c-7022-4f6b-af52-022bbfb36e2f"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/1024/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Second Animal Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#323232",
                            }
                          }
                        >
                          Default
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-Second Mechanical Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/b045ca24-7424-47dd-941a-1969013eb521"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/111/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Second Mechanical Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#323232",
                            }
                          }
                        >
                          Default
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-Second Party Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/2f437f15-a652-413b-8f9a-c690f71ac18f"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/158/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Second Party Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#323232",
                            }
                          }
                        >
                          Default
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-Wedding Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/ca1b452b-2968-44d1-972d-ed86b9721f49"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/1013/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Wedding Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#323232",
                            }
                          }
                        >
                          Default
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-Sport Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/017c4359-9387-4788-9650-318c8042348a"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/1023/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        Sport Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#323232",
                            }
                          }
                        >
                          Default
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
              <div
                className="CollectionCard-spacing_unselected-19"
                data-testid="Home-Collection-Card-People Collection"
              >
                <div
                  className="CollectionCard-container-20"
                  data-testid="Home-Collection-Card-Container"
                >
                  <a
                    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                    href="gallery/5b825d2b-f71d-4f20-accd-3b84f4d10093"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    style={
                      Object {
                        "textDecoration": "none",
                      }
                    }
                  >
                    <img
                      alt=""
                      className="CollectionCardImage-layout-21"
                      data-testid="Collection-Card-Image"
                      height={200}
                      src="https://picsum.photos/id/375/200"
                      width="350"
                    />
                    <div
                      className="CollectionCardText-text_wrapper-23"
                      data-testid="Collection-Card-Text-Wrapper"
                    >
                      <img
                        alt=""
                        className="CollectionCardText-icon-24"
                        data-testid="Collection-Card-Image-Icon"
                        height={20}
                        src="collection-image-o.png"
                        width={20}
                      />
                      <p
                        className="CollectionCardText-text-25"
                        data-testid="Collection-Card-Title"
                      >
                        People Collection
                      </p>
                      <div
                        className="CollectionCardText-tag-26"
                      >
                        <div
                          className="HomeCollectionTag-layout-27"
                          data-testid="tag-layout"
                          style={
                            Object {
                              "background": "#323232",
                            }
                          }
                        >
                          Default
                        </div>
                      </div>
                    </div>
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </main>
    </div>
  </div>
</main>
`;
